tox.ini

```diff
@@ -27,6 +27,7 @@ envlist =
     {py27,py34,py35,py36}-tracer
     {py27,py34,py35,py36}-integration
     {py27,py34,py35,py36}-ddtracerun
+    {py27,py34,py35,py36}-opentracer
```

Comment:
(Also) you need to update the `.circleci/config.yml` file otherwise these tests are not executed in the CI. This is a terrible thing that we need to address. Hopefully, our system detects that and we can safely avoid to merge patches that require this change.